run.py

```diff
@@ -1,80 +1,115 @@
 #!/usr/bin/env python3
 
+import logging
 import os
-from pathlib import Path
+import re
 import subprocess as sp
 import sys
+from pathlib import Path
 
 import flywheel_gear_toolkit
-from flywheel_gear_toolkit.utils import file
-from flywheel_gear_toolkit.utils import zip_tools
+from flywheel_gear_toolkit.utils import file, zip_tools
+from pathvalidate import sanitize_filepath
+
+
+def create_sanitized_filepath(filepath):
+    """
+    Return symlink to filepath without illegal characters.
+
+    Illegal characters are not alphanumeric, '.', '_', or '-'.
+
+    Args:
+        filepath (Path-Like): Filepath with potential invalid characters
+
+    Returns:
+        str: Path to create symbolic link to original file.
+    """
+    try:
+        log
+    except NameError:
+        log = logging.getLogger(__name__)
+    filepath = re.sub(r"(t2 ?_?)\*", r"\1star", str(filepath), flags=re.IGNORECASE)
+    sanitized_filepath = sanitize_filepath(filepath, platform="auto").replace(" ", "_")
+    if filepath != sanitized_filepath:
+        log.info("Linking %s to %s.", filepath, sanitized_filepath)
+        os.symlink(filepath, sanitized_filepath)
+
+    return sanitized_filepath
 
 
 def fail_check(context):
-    
+
     input_path = context.get_input_path("Input_file")
     expected_extension = ".svs"
-    
+
     try:
-        passes = file.is_valid(input_path)#, expected_extension)
+        passes = file.is_valid(input_path)  # , expected_extension)
     except FileNotFoundError as e:
         context.log.exception(e)
-        raise(e)
+        raise (e)
```

Comment:
This is the medling work of black.  Does this change the behavior?  `raise(e)` returns e.  does` raise (e)` return a tuple `(e)`?  please verify.  Arguably my fault for still insisting on including parenthesis, but they lulled me into a false sense of security with print statements in python2 and I won't be hurt again, Josh.  I won't be hurt again.

Reply:
FURTHER it makes sense to continue to use parenthesis.  Everything else uses parenthesis.  Logging, function calls, print statements.  Raises and returns are inconsistent with the rest of python syntax.  WHY WOULD THEY DO THIS?  IT DOESN'T MAKE SENSE, I TELL YOU.  it doesn't make sense.